pkgs/by-name/gp/gprename/package.nix

```
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  # build dependencies
+  makeWrapper,
+  moreutils,
+  # runtime dependencies
+  perlPackages,
+  gettext,
+  glib,
+  gtk3,
+  gobject-introspection,
+  pango,
+  harfbuzz,
+  gdk-pixbuf,
+  at-spi2-atk,
+}:
+stdenv.mkDerivation (prev: {
```

```suggestion

stdenv.mkDerivation (finalAttrs: {
```

Let's keep it similar to existing code.